<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_to_db</class>
 <widget class="QDialog" name="add_to_db">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add project to database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblProjnb">
       <property name="text">
        <string>Project Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEProjnb"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblProjName">
       <property name="text">
        <string>Project Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEProjName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblMainDir">
       <property name="text">
        <string>Main Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LEMainDir"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lblLocalDirs">
       <property name="text">
        <string>Local Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LELocalDirs"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="addToDbButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addToDbButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>add_to_db</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addToDbButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>add_to_db</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
